api/src/server.ts

```diff
@@ -38,12 +55,14 @@ const envToLogger = {
     serializers: {
       req: requestSerializer
```

Comment:
```suggestion
      req: (req: FastifyRequest) => {
        return {
          method: req.method,
          url: req.url
        };
      }
```

The `requestSerializer` is way too noisy for local development. Parity is not a concern here.